org-brain.el

```diff
@@ -79,6 +79,19 @@ Example: \"<--\" would add \"<--A\" in the example above."
   :type '(restricted-sexp :match-alternatives
            (integerp 't 'nil)))
 
+(defcustom org-brain-backlink-heading nil
+  "If the org file heading should be used when creating the backlink.
+
+Example: If you create a brain-link in A to B and A is an org file with the headings:
+* Parent header
+** Child
+[brain:linkToB]
+
+Setting this variable to t will create the following backlink in B:
+[[file:A.org::*Child][Parent header > Child]]."
```

Comment:
> Or is it to get a better context?

Yes, that's it. Otherwise, in a situation where I have notes on binary trees like this:

```
* Computer science book
** Trees
``` 
Creating a back link like `> Trees` would remove important context from it. I can see that it may not be useful to some people though. That's the reason I added the `org-brain-backlink-heading` option. What do you think? I can remove it if you prefer.